From 354389c93e26bb8d8e8c1c126b01d838a6a214c8 Mon Sep 17 00:00:00 2001
From: Roy Li <rongqing.li@windriver.com>
Date: Sat, 15 Feb 2014 09:45:00 +0800
Subject: [PATCH] policy/modules/roles/sysadm: allow sysadm to run rpcinfo

Fixes:
$ rpcinfo
rpcinfo: can't contact rpcbind: RPC: Remote system error - Permission denied

avc:  denied  { connectto } for  pid=406 comm="rpcinfo"
path="/run/rpcbind.sock" scontext=root:sysadm_r:sysadm_t
tcontext=system_u:system_r:rpcbind_t tclass=unix_stream_socket
permissive=0

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Roy Li <rongqing.li@windriver.com>
Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/roles/sysadm.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/roles/sysadm.te b/policy/modules/roles/sysadm.te
index f0370b426..fc0945fe4 100644
--- a/policy/modules/roles/sysadm.te
+++ b/policy/modules/roles/sysadm.te
@@ -962,6 +962,7 @@ optional_policy(`
 ')
 
 optional_policy(`
+	rpcbind_stream_connect(sysadm_t)
 	rpcbind_admin(sysadm_t, sysadm_r)
 ')
 
-- 
2.17.1

